### PR TITLE
Define command callback as a named function

### PR DESCRIPTION
**Pull request by nukosuke**
- Made lambda callback into a named function to avoid too many nesting

### CODE REVIEW

File: command/apply.go [hunk 1]
```diff
@@ -24,141 +24,136 @@ var applyCommand = &cobra.Command{
 	Use:   "apply",
 	Short: "Apply configuration to Zendesk",
 	Long:  "Apply configuration to Zendesk",
-	Run: func(cmd *cobra.Command, args []string) {
```
Comment (nukosuke) [inline command/apply.go]: here

File: command/apply.go [hunk 2]
```diff
@@ -24,141 +24,136 @@ var applyCommand = &cobra.Command{
...
-		exe := NewStepExecutor()
-		zd, _ := zendesk.NewClient(nil)
-		zfconfig := &config.ZenformConfig{}
-		currentState := NewZenformState()
-		var conf config.Config
+	Run:   applyFunc,
```
Comment (nukosuke) [inline command/apply.go]: here

File: command/apply.go [hunk 3]
```diff
@@ -24,141 +24,136 @@ var applyCommand = &cobra.Command{
...
+				exe.Error(err.Error())
 				return err
 			}
+			exe.Success("OK")
 			return nil
 		})
-		// TODO:
-		// - make request
-		// - request Zendesk API
-		// - parse response
```
Comment (hashijun) [inline command/apply.go]: Are these TODO are all solved?

Comment (nukosuke) [reply command/apply.go]: Yes, these features have been implemented in each step, stepCreateTicketFields, stepCreateTicketForms and stepCreateTriggers.

Comment (hashijun) [reply command/apply.go]: 👍

**Comment by nukosuke** [comment]
THX!!

**Pull request merged**